Extensions/CMake/SlicerExtensionDescriptionSpec.cmake

```diff
@@ -203,7 +203,7 @@ function(slicer_extension_description_spec_defaults_test)
     ENABLED
     STATUS
     )
-  foreach(name IN LISTS required optonal)
+  foreach(name IN LISTS required optional)
```

Comment:
Was this a bug?

Reply:
Looks like it was a bug since there's a list named `optional` defined just above.  @jcfr could you look?  Did this have any effect?

Reply:
Yikes, sorry, didn't mean to change source code.

Reply:
Good catch :ok_hand: 

This is indeed an issue, I will submit an independent pull request including the relevant `Co-authored-by:` trailers.

Reply:
This is being addressed by https://github.com/Slicer/Slicer/pull/6485